src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -80,6 +85,9 @@ export const FilesTableRow = ({
   showAllColumns,
 }: FilesTableRowProps) => {
   const { t } = useTranslation();
+  const { identifier } = useParams<IdentifierParams>();
```

Comment:
```suggestion
  const { identifier as registrationIdentifier } = useParams<IdentifierParams>();
```

Ikke så viktig, men kan være kjekt å spesifisere skille mellom registration- og file-identifier. (mtp `await deleteFile(identifier, file.identifier);` lenger nede.

Reply:
Tenkte samme selv så prøvde men det gikk ikke, så da bare droppa jeg det bare fordi dette så så rart ut 😅 
```
const registrationIdentifier = useParams<IdentifierParams>().identifier;
```

Men når jeg sjekka nå ser jeg at dette kan gjøres slik, så skal fikse det"
```
 const { identifier: registrationIdentifier } = useParams<IdentifierParams>();
```

Reply:
Fant ut at dette brukes for wizard for importkandidater også, så gikk tilbake til `identifier` sånn at det passer for begge